manual_correction.py

```diff
@@ -328,6 +354,29 @@ def generate_qc(fname, fname_label, task, fname_qc, subject, config_file):
     print("Archive created:\n--> {}".format(fname_qc + '.zip'))
 
 
+def denoise_image(fname):
+    """
+    Denoise image using non-local means adaptative denoising from P. Coupe et al. as implemented in dipy. For details,
+    run sct_maths -h
+    :param fname:
+    :return:
+    """
+    print("Denoising {}".format(fname))
+    fname_denoised = utils.add_suffix(fname, '_denoised-p1b2')
+    os.system('sct_maths -i {} -denoise p=1,b=2 -o {}'.format(fname, fname_denoised))
```

Comment:
Do you think it would be better to provide the options for denoising as well? My concern is - would it be good to assume that `p=1, b=2` will be good for all kinds of images? Note that the default options in SCT docs are `p=1, b=5`.

Reply:
good point-- but for now, this option is _pretty_ niche (ie: few people use this script, and even fewer ppl use this flag). And we want _b_ to be small because lesions could be missed (few people actually understand the meaning of this "b" parameter, and I suspect they won't read the paper, so better to impose a "safe" value than letting them choose a wrong one)